src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -40,6 +40,18 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
         return Index.fromOneBased(Integer.parseInt(trimmedIndex));
     }
 
+    /**
+     * Parses an {@code intString} into an {@code Integer} and returns it.
+     * @throws ParseException if the string cannot be converted into an integer.
+     */
+    public static int parseInt(String intString) throws ParseException {
+        String trimmedInt = intString.trim();
+        if (trimmedInt.isEmpty() || !StringUtil.isNonZeroUnsignedInteger(trimmedInt)) {
+            throw new ParseException(MESSAGE_INVALID_INDEX);
+        }
+        return Integer.parseInt(trimmedInt);
+    }
```

Comment:
This was actually for my Sample Command, so it does nothing now. It may be useful in the future, though.